scripts/build-docker

```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env bash
 
+ARCH=$(go env GOARCH)
```

Comment:
./release script should still happen from one machine that is amd64. And only for building docker images we can run on target arch slaves. e.g ppc64le, arm etc..